Cargo.toml

```diff
@@ -603,7 +603,7 @@ solana-rpc-client-api = { path = "rpc-client-api", version = "=2.2.0" }
 solana-rpc-client-nonce-utils = { path = "rpc-client-nonce-utils", version = "=2.2.0" }
 solana-runtime = { path = "runtime", version = "=2.2.0" }
 solana-runtime-transaction = { path = "runtime-transaction", version = "=2.2.0" }
-solana-sbpf = "=0.9.0"
+solana-sbpf = { git = "https://github.com/anza-xyz/sbpf.git", branch = "main" }
```

Comment:
🤔

Reply:
Yep, that is procedure. First test if CI runs through on the main branch, then only cut the SBPF release.